src/classes/REL_Relationships_TDTM.cls

```diff
@@ -72,40 +87,34 @@ public class REL_Relationships_TDTM extends TDTM_Runnable {
         //Keeps track of relationships that have already been deleted to prevent Entity is Deleted error
         Set<Id> hasBeenDeleted = new Set<Id>();
 
-        if (triggerAction == TDTM_Runnable.Action.afterUpdate) {
-            hasRun = true;
-        }
-
-        if (triggerAction == TDTM_Runnable.Action.afterDelete) {
+        if (triggerAction == TDTM_Runnable.Action.AfterDelete) {
             hasBeenDeleted.addAll((new Map<Id, Relationship__c>(newList)).keySet());
         }
         
         Integer i = 0;          // processing counter            
-        for(Relationship__c r : newList) {
+        for (Relationship__c r : newList) {
             // AFTER INSERT
-            if (triggerAction == TDTM_Runnable.Action.afterInsert) {
+            if (triggerAction == TDTM_Runnable.Action.AfterInsert) {
                 if (r.ReciprocalRelationship__c == null) {
                     relationshipsToCreate.add(r);
-                }
-                if (r.ReciprocalRelationship__c != null) {
+                } else {
                     reciprocalsToUpdate.add(r);
                 }
             }
             
             // AFTER UPDATE
-            if (triggerAction == TDTM_Runnable.Action.afterUpdate) {   
+            if (triggerAction == TDTM_Runnable.Action.AfterUpdate) {   
                 if (r.ReciprocalRelationship__c == oldList[i].ReciprocalRelationship__c 
-                && r.RelatedContact__c != null) {
+                        && r.RelatedContact__c != null) {
```

Comment:
how do you feel about this format:
```
if (r.ReciprocalRelationship__c == oldList[i].ReciprocalRelationship__c &&
    r.RelatedContact__c != null) {
```

no big deal and probably all about the same in the end. I've usually put my && at the end of a line so when I see them hanging, I know there's more. Either works though :)

Reply:
I can't say I like the idea of putting logical operators at the end of the line. It is probably fine here but once the logic getting more complicated, it will be really hard to read...

Reply:
I moved it back to the end of the line. I don't really have a preference either way.